### ISSUE TITLE
JEI sieve bug when you have too many items added

### ISSUE THREAD

**Issue by Onigeshi** [Jozufozu/ExNihiloOmnia#97]
I've started adding all the seeds I have to get them from sieving the dirt, now when I go to JEI to check them this is what I get: http://prntscr.com/dbqbqo 
The old mod was creating a new page when there was too many items to show like in this case, also from what I see your mod is separating the items in 2 pages, 1 for the items added by the mod and 1 for the items added by us into the config.. wouldnt be better if all would show in the same page and create a second page only if there are too many items ?

**Comment by Jozufozu** [comment]
Yeah... I thought that might happen but never thought it would, I'll see what I can do
